src/components/course/data/courseLoader.ts

```diff
@@ -115,7 +125,7 @@ export default function makeCourseLoader(queryClient) {
         // If learner is an assignment-only learner and is not assigned to the currently
         // viewed course, redirect to the Dashboard page route.
         if (isAssignmentOnlyLearner && !isCourseAssigned) {
-          throw redirect(generatePath('/:enterpriseSlug', { enterpriseSlug }));
+          throw redirect(generatePath('/:enterpriseSlug', { enterpriseSlug: enterpriseSlug as string }));
```

Comment:
[nit] some remaining test data

Reply:
Unlike the other instance, this one can be removed, though. The `CourseRouteParams` coerces the `enterpriseSlug` to only be string already.